rbac/management/workspace/utils.py

```diff
@@ -110,3 +111,15 @@ def _get_group_list_from_resource_definitions(resource_definitions: dict) -> lis
         except (ValueError, TypeError):
             group_list.remove(gid)
     return group_list
+
+
+def check_total_workspace_count_exceeded(request) -> bool:
+    """Check if the current org has exceeded the allowed amount of workspaces.
+
+    Returns True if total number of workspaces is exceeded.
+    """
+    org = request.tenant
+    max_limit = settings.WORKSPACE_ORG_CREATION_LIMIT
+
+    workspace_count = Workspace.objects.filter(tenant=org).count()
```

Comment:
🤔 I would like to double check that we want to limit the number of all workspaces or standard workspaces? @lpichler @astrozzc

Reply:
@EvanCasey13 let's use standard workspaces